### PR TITLE
chore: renamed to connect, added some minor QoL improvements on cashout

### PR DESCRIPTION
**Pull request by Hugo0**
- renamed to connect
- added some minor QoL improvements on cashout

a lot of the diff is just because of prettier formatting ( we should be more consistent in everyone using it)

### CODE REVIEW

File: src/components/Cashout/Components/Initial.view.tsx [hunk 1]
```diff
@@ -393,7 +393,7 @@ export const InitialCashoutView = ({
                 }
             >
                 {!isConnected ? (
-                    'Create or Connect Wallet'
```
Comment (nezz0746) [inline src/components/Cashout/Components/Initial.view.tsx:396]: Wanted to do that as well, shortens elements in the header

File: src/components/Cashout/Components/Initial.view.tsx [hunk 2]
```diff
@@ -56,10 +56,17 @@ export const InitialCashoutView = ({
...
+    // TODO: @Hugo0 value is ambigous with price - it should be tokenAmount and tokenPrice. But this means changes across a bunch of files.
     const [_tokenValue, _setTokenValue] = useState<string | undefined>(
         inputDenomination === 'TOKEN' ? tokenValue : usdValue
     )
 
+    useEffect(() => {
+        if (!_tokenValue) {
+            _setTokenValue('0')
+        }
+    }, [])
```
Comment (Hugo0) [inline src/components/Cashout/Components/Initial.view.tsx:68]: intention: start off with 0 as selected amount to trigger min cashout message.
Maybe there's bettter way to do this

File: src/components/Cashout/Components/Initial.view.tsx [hunk 3]
```diff
@@ -77,13 +84,23 @@ export const InitialCashoutView = ({
...
 
+    const isDisabled = useMemo(() => {
+        return (
+            !_tokenValue ||
+            (!selectedBankAccount && !newBankAccount) ||
+            !xchainAllowed ||
+            !!isBelowMinLimit ||
+            !!isExceedingMaxLimit
+        )
+    }, [_tokenValue, selectedBankAccount, newBankAccount, xchainAllowed, isBelowMinLimit, isExceedingMaxLimit])
```
Comment (Hugo0) [inline src/components/Cashout/Components/Initial.view.tsx:102]: intention: fixes bug where the proceeed button wouldn't get enabled even if i added a new account, and seems like better logic

File: src/components/Cashout/Components/Initial.view.tsx [hunk 4]
```diff
@@ -174,9 +191,12 @@ export const InitialCashoutView = ({
...
             }
             onNext()
-        } catch (error) {
+        } catch (error: any) {
             console.error('Error:', error)
-            setErrorState({ showError: true, errorMessage: 'An error occurred. Please try again.' })
+            setErrorState({
+                showError: true,
+                errorMessage: error.message || 'An error occurred. Please try again.',
+            })
```
Comment (Hugo0) [inline src/components/Cashout/Components/Initial.view.tsx:199]: intention: this propagates the error message like insufficient funds etc. Much more informative.

File: src/utils/cashout.utils.ts [hunk 5]
```diff
@@ -166,8 +166,8 @@ export async function awaitStatusCompletion(
         status = statusData[`${type}_status`]
 
         if (status === 'under_review') {
-            if (type === 'tos') throw new Error('TOS is under review')
-            else if (type === 'kyc') throw new Error('KYC is under review.')
+            if (type === 'tos') throw new Error('TOS_UNDER_REVIEW')
+            else if (type === 'kyc') throw new Error('KYC_UNDER_REVIEW')
         } else if (status !== 'approved') {
             await new Promise((resolve) => setTimeout(resolve, 5000)) // wait 5 seconds before checking again
         }
```
[the remark this answers is not in the document]
Comment (Hugo0) [reply src/utils/cashout.utils.ts:165]: v good point

**Pull request merged**